tests/integration/element__waiting_search_on_actions_like_click_test.py

```diff
@@ -33,16 +33,16 @@ def test_waits_for_visibility_minimum_needed_time(session_browser):
         <a href="#second" style="display:none">go to Heading 2</a>
         <h2 id="second">Heading 2</h2>'''
     ).execute_script_with_timeout(
-        'document.getElementsByTagName("a")[0].style = "display:block";', 500
+        'document.getElementsByTagName("a")[0].style = "display:block";', 0.5
     )
     element = session_browser.element('a')
     # stamp_before = time.time_ns()
 
     element.click()
 
     # stamp_after = time.time_ns()
-    # deviation_ms = 200
-    # assert stamp_after - stamp_before < (500 + deviation_ms) * 1000000
+    # deviation_sec = 0.2
+    # assert stamp_after - stamp_before < (0.5 + deviation_sec) * 1000000
```

Comment:
Here was milliseconds to nanoseconds conversion. Now since it has become seconds to nanoseconds conversion - need to multiply * 1000 more.
```suggestion
    # deviation_sec = 0.2
    # assert stamp_after - stamp_before < (0.5 + deviation_sec) * 1000000000
```

or
```suggestion
    # deviation_sec = 0.2
    # assert stamp_after - stamp_before < (0.5 + deviation_sec) * pow(10, 9)
```

Reply:
Thanks, already done.